Cython/Shadow.pyi

```diff
@@ -99,4 +299,4 @@ class typedef(CythonType, Generic[_T]):
 #class _FusedType(CythonType, Generic[_T]):
 #    def __init__(self) -> None: ...
 
-#def fused_type(*args: Tuple[_T]) -> Type[FusedType[_T]]: ...
+def fused_type(*args: Any) -> Any: ...
```

Comment:
This change is worth a comment. Otherwise, readers will be tempted to improve it.
Why doesn't the return type at least say `Type[Any]`?